demos/toolbar/index.php

```diff
@@ -128,14 +128,14 @@
 
 		<div data-demo-html="true">
 			<div data-role="header">
-				<a href="#" class="ui-button-left ui-button ui-button-inline ui-mini ui-corner-all">View<span class="ui-icon ui-icon-grid"></span></a>
+				<a href="#" class="ui-toolbar-header-button-left ui-button ui-button-inline ui-mini ui-corner-all ui-icon-beginning">View<span class="ui-icon ui-icon-grid"></span></a>
```

Comment:
All these icon postion classes need to be removed the icon should be inline where it belongs with a spce between it and the text